### DISCUSSION TITLE
I created a few packs, now I can't find the links.

### DISCUSSION THREAD

**Discussion by seefood** [signalstickers/signalstickers#858] [Q&A] [answered]
I retroactively realized the links only appear once when I  create the pack, So to register the pack I have to create it again? how does that make sense? all the data is stored in my client and each time I post a sticker in a chat, so why can't I get the URL to post online at a later point?

**Comment by romainricard** [comment, marked as answer]
Hey 👋 the link does not appear on Signal desktop, but you can use the "Share" icon in your pack list on Android/iOS, and this will allow you to get the link.
